tico/passes/convert_to_relu6.py

```diff
@@ -58,7 +59,7 @@ def convert(self, exported_program, node):
         input = args.input
 
         with graph.inserting_after(node):
-            relu_node = graph.call_function(torch.ops.aten.relu6.default, args=(input,))
+            relu_node = create_node(graph, torch.ops.aten.relu6.default, args=(input,))
```

Comment:
For some nodes, origins are missing. Do you mean it?

Reply:
Ah, right. I intentionally didn't set `origin` because those are propagated with `replace_all_uses_with`.